core/server/data/migrations/versions/3.33/01-add-email-recipients-tables.js

```diff
@@ -0,0 +1,34 @@
+const logging = require('../../../../../shared/logging');
+const commands = require('../../../schema').commands;
+
+module.exports = {
+    async up({connection}) {
+        // table creation order is important because of foreign key constraints,
+        // email_recipients references email_batches so email_batches has to exist when creating
+        return Promise.each(['email_batches', 'email_recipients'], async (table) => {
+            const tableExists = await connection.schema.hasTable(table);
+
+            if (tableExists) {
+                return logging.warn(`Adding table: ${table}`);
```

Comment:
Can you change the logging message here to something like `Not adding ${table} - table already exists`? :slightly_smiling_face: